templates/definition/meter/kostal-piko-hybrid.yaml

```diff
@@ -13,6 +13,11 @@ params:
     advanced: true
 render: |
   type: custom
```

Comment:
Das ist immer der gleiche Wert, egl welche Usage??

Reply:
Um ehrlich zu sein weis ich nicht mal was das bedeutet. Ich bin leider kein gelernter Programmierer und kann nur Kleinigkeiten selber lösen. Und das was du markiert hast war schon vor meiner Änderung im template.

Reply:
Hast du die Änderungen denn getestet und die Werte validiert?

Reply:
Ja alles was ich geändert habe, wurde getestet und wurde auch mit den richtigen Werten in der UI (Konfiguration) angezeigt.

Reply:
Das ist super- und hast du bei den Kostals auch alle Varianten grid/pv/wo passend battery getestet? Denn Zählerstand hast du nur 1x konfiguriert, das müssten aber 3 unterschiedliche Werte sein?

Reply:
Ich selber habe keine Speicher bei den Kostals darum ich das nur bei PV ergänzt und getestet.

Reply:
Leider nicht. Bei Kostal hast Du es überall ergänzt (ausserhalb der "if usage" statements- das muss da aber rein). Das kann so eigentlcih nicht funktionieren. Hoymiles sieht gut aus.